core/field.ts

```diff
@@ -1217,19 +1228,20 @@ export abstract class Field implements IASTNodeLocationSvg,
    */
   setMarkerSvg(markerSvg: SVGElement) {
     if (!markerSvg) {
```

Comment:
I think the parameter to this function needs to be nullable, otherwise this check doesn't make sense. some of our other `setMarkerSvg` methods have the same logic and the parameter is nullable there. I wonder if this was created from our closure code where we didn't always use `?` consistently